### ISSUE TITLE
Are the SCAD models with the trackball?

### ISSUE THREAD

**Issue by xaionaro** [Schievel1/dactyl_manuform_r_track#2]
Hello.

I see that the SCAD model `right.scad` does not have the trackball socket:
![Screenshot from 2022-06-26 14-43-06](https://user-images.githubusercontent.com/2349376/175817114-7b04c521-6b74-4ffb-b886-83bbb9ed992b.png)

And I'm wondering if there is a model with the socket :)

**Comment by Schievel1** [comment]
No, sadly not. You can try making some sense out of the files in the original repo, as I couldn't: https://gitlab.com/keyboards1

Or you can use the STL and load it into Openscad. Then you could not really change the shape but you can add or remove additional forms to it. That's what I did to change the holes for the USB adapter.

**Comment by xaionaro** [comment]
I see. Thanks.

I decided to load an STL of a Dactyl configuration I want into Blender and add a trackball socket there :)
